include/gmapping/gridfastslam/gridslamprocessor.h

```diff
@@ -15,6 +15,8 @@
 #include <gmapping/scanmatcher/scanmatcher.h>
 #include "motionmodel.h"
 
+#include <omp.h>
```

Comment:
That's necessary because I'm calling the`omp_get_wtime()` function to measure execution time.